network-api/networkapi/wagtailpages/models.py

```diff
@@ -1435,4 +1435,10 @@ class YoutubeRegretsPage(FoundationMetadataPageMixin, Page):
         StreamFieldPanel('regret_stories'),
     ]
 
+    zen_nav = True
```

Comment:
The youtube page really should not need to care about this flag, its HTML etc. is bespoke. If we want the styling associated with the zen nav, then we should hardcode that into the template, not add a model migration.